grafana/dashboards/trip.json

```diff
@@ -1,4 +1,70 @@
 {
+  "__inputs": [
```

Comment:
I'm not sure where these changes are coming from. It would be nice if you could remove them.

Reply:
Aaaah, I think those strange changes came from switching this on
![image](https://user-images.githubusercontent.com/8087682/95010792-79663080-062c-11eb-8fcf-d7e21e17c8fb.png)

I will make another PR...